src/gui/static/src/app/components/layout/hardware-wallet/hw-passphrase-dialog/hw-passphrase-dialog.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163608)
```suggestion
import { Component, OnInit, OnDestroy } from "@angular/core";
```